src/app/talent/talent/talent.component.ts

```diff
@@ -38,24 +42,70 @@ export class TalentComponent extends AbstractComponent<Talent> {
 	}
 
 	public onFileChange(event) {
-		const reader = new FileReader();
-
-		this.file = event.target.files[0];
-		reader.readAsDataURL(event.target.files[0]);
-		reader.onload = () => {
-			this.originalImage = reader.result;
+		if (event.target.name === 'listingImage') {
+			this.listingImage = event;
+			this.listingFile = event.target.files[0];
+		} else {
+			this.profileImage = event;
+			this.profileFile = event.target.files[0];
 		}
 	}
 
+	public onListingCropped(event) {
+		this.listingCroppedImage = event.base64;
+	}
+
+	public onProfileCropped(event) {
+		this.profileCroppedImage = event.base64;
+	}
+
 	public save() {
-		const image: FormFile = {
-			file: this.file,
-			key: 'storeImage'
-		};
+		fetch(this.profileCroppedImage)
+		.then(result => result.blob())
+		.then((blob) => {
+			const file: File = new File([blob], 'profileCropped', {
+				type: 'image/jpeg'
+			});
 
-		this.service.createForm(this.item, [image]).subscribe((data) => {
-			console.log(data);
+			const profile: FormFile = {
+				file: this.profileFile,
+				key: 'profileImage'
+			};
+
+			const profileCropped: FormFile = {
+				file: file,
+				key: 'profileCroppedImage'
+			};
+
+			console.log(profile);
+			console.log(profileCropped);
+
+			fetch(this.listingCroppedImage)
```

Comment:
move the fetch logic into a different function, and call it for each file you need it while using Promise.all to wait for the response from both files. we can talk more about this one